src/bindings/python/docs/test_examples.md

```diff
@@ -35,6 +45,31 @@ To run full test suite one can utilize `tox` command:
 tox
 ```
 
+### Check codestyle of Python API
+There are two packages used in the project to check codestyle of python code: *mypy* and *flake8*.
+Additionally OpenVINO™ uses the custom configuration file to exclude some strict rules.
+
+For checking the codestyle of Python API 2.0 execute this command:
+```
+python -m flake8 ./src/openvino/ --config=setup.cfg
+python -m mypy ./src/openvino --config-file ./setup.cfg
+```
+For checking the codestyle of compatibility nGraph Python API execute this command:
+```
+python -m flake8 ./src/compatibility/ngraph/ --config=setup.cfg
+python -m mypy ./src/compatibility/ngraph --config-file ./setup.cfg
+```
+For checking the codestyle of compatibility InferenceEngine Python API execute this command:
+```
+cd src/compatibility/openvino
+python -m flake8 ./ --config=setup.cfg
+python -m mypy ./ --config-file ./setup.cfg
+```
+Also we suggest to run the codestyle check above tests if some new were added.
```

Comment:
```suggestion
It's recommended to run the codestyle check tests above when new APIs are added.
```

Reply:
no, it meant `if new tests have been added, run the codestyle check for the tests folder`

Reply:
would this be correct?
It's recommended to run the mentioned codestyle checks whenever new tests are added.

Reply:
@kblaszczak-intel thanx,
will use your phrase